Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
+            case print
+            case nslog
+        }
+
+        public var minLevel = Log.Level.error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+        public var output = ConsoleOutput.print
```

Comment:
I disagree with not using type inference, unless for compile time problems which I'm not aware.
Otherwise I **always** use type inference, and I would suggest the same.
Apple suggests the same, as others code style guides:
[RayWenderlich](https://github.com/raywenderlich/swift-style-guide#type-inference)
[Hypersolo](https://github.com/hyperoslo/iOS-playbook/blob/master/style-guidelines/Swift.md#type-inference)